src/OSDP.Net/Bus.cs

```diff
@@ -306,6 +304,16 @@ private async Task StartPollingAsync()
                     delayTime.WaitOne(IdleLineDelay(2));
                 }
             }
+
```

Comment:
I moved the Close of the connection to within the poll loop. Makes sense, since the poll loop calls Open, it can also call Close. And since there's only one poll loop running for each connection, there will not be multiple calls to Close.